package.json

```diff
@@ -36,6 +36,10 @@
     "clarify": "^1.0.5",
     "dependency-check": "^2.5.1",
     "ghooks": "^1.2.1",
+    "lodash.filter": "^4.0.1",
+    "lodash.map": "^4.0.1",
+    "lodash.trim": "^4.0.1",
+    "lodash.trimstart": "^4.4.0",
```

Comment:
I would prefer using directly the `lodash` package as it simplifies code dedupe and packages maintenance.

It's still possible to import only the needed methods: `import forEach from 'lodash.forEach`' (and it's a good idea when compiling code for the web).